docs/troubleshooting-oauth2.md

```diff
@@ -100,13 +108,34 @@ This issue occurs when the token does not have enough scopes or permissions to a
+4. Click on the tree option which matches the URL of the management UI
 5. Select the Key *rabbitmq.credentials* in the right panel
-6. Copy its value 
+6. Copy its value
 7. Go to the url https://jwt.io
 8. Paste the value into the text field *Encoded*
-9. Look at the payload's text field *Decoded* 
-10. Search for the token attribute `scope` in the tokens' payload or for the value configured in `auth_oauth2.additional_scopes_key`, if any. 
-11. Once you found the appropriate token's scope attribute, find within the attribute's value any of the scopes listed above. Please, take into account if you have configured [auth_oauth2.scope_prefix](./oauth2#scope-prefix). If you did, the scopes will be named like  `myprefix_tag:administrator`. If you instead configured [scope aliases](./oauth2-examples#use-scope-aliases), you need to find the scope alias that maps to one of the scopes listed above. 
+9. Look at the payload's text field *Decoded*
+10. Search for the token attribute `scope` in the tokens' payload or for the value configured in `auth_oauth2.additional_scopes_key`, if any.
+11. Once you found the appropriate token's scope attribute, find within the attribute's value any of the scopes listed above. If [auth_oauth2.scope_prefix](./oauth2#scope-prefix) is used, it must be taken into account: the scopes will be named like  `myprefix_tag:administrator`. If [scope aliases](./oauth2-examples#use-scope-aliases) are used, find the scope alias that maps to one of the scopes listed above
+
+
+### OpenId Discovery Endpoint not Reachable Due to a Certificate SAN (or CN) and Target Hostname Mismatch {#openid-discovery-endpoint-san-cn-mismatch}
+
+This issue is not necessarily specific to the management UI, it may also occur when an application is authenticating via one of the messaging protocols.
+It occurs when RabbitMQ has to download the OpenID Connect configuration via the URL configured in `auth_oauth2.issuer` and the certificate used by the issuer uses a wildcard certificate.
+This means that the certificate's SAN (Subject Alternative Name) or CN certificate attributes does not match the issuer's domain name exactly.
+This is a very common situation in SaaS deployments.
+
+#### Steps to reproduce the issue
+
+1. Open the root URL of the management UI in the browser.
+2. Click on "Click here to login".
+3. Enter your credentials.
+4. You are redirected back to RabbitMQ but with an error "No Authorized"
+
+#### Troubleshoot the issue
 
+1. Access RabbitMQ [logs](./logging)
+2. Look up for `{bad_cert,hostname_check_failed}`
+3. Look for an entry it says that RabbitMQ tried to contact the url found in `auth_oauth2.issuer` and the issuer presented a wildcard certificate
+4. To fix this issue, add the following line to your RabbitMQ configuration `auth_oauth2.https.hostname_verification = wildcard`. Or `auth_oauth2.oauth_providers.<my_oauth_provider_name>.https.hostname_verification = wildcard` if you are using multiple OAuth providers
```

Comment:
can't start a sentence with "Or", just run the sentence on from the previous one, i.e. ".....wildcard, or, ...."